### DISCUSSION TITLE
Cloudflare custom domain - why is this not part of docs?

### DISCUSSION THREAD

**Discussion by xpluscal** [GitbookIO/community#362] [0. General bugs and questions] [answered]
It seems to be a common issue that cloudflare custom domain CNAME doesn't work with Gitbook.
How can a user solve this?

Can this be part of the docs?

**Comment by rodrcastro** [comment, marked as answer]
Hey @xpluscal 👋 You should be able to add custom domains registered with Cloudflare normally.

The only step we ask for attention when setting up with Cloudflare is to not use the proxy option - and [that is in our docs](https://docs.gitbook.com/publishing/custom-domain/configure-dns#are-you-using-cloudflare).

If you are facing different issues, I suggest contacting our support team (support@gitbook.com). Let them know which space or organization you are trying to use the custom domain in, what domain you are trying to use, and the error you are receiving. They should be able to help from there :)
